src/streaming/controllers/ProtectionController.js

```diff
@@ -85,7 +74,7 @@ MediaPlayer.dependencies.ProtectionController = function () {
                 for (ksIdx = 0; ksIdx < supportedKS.length; ksIdx++) {
                     if (this.keySystem === supportedKS[ksIdx].ks) {
 
-                        requestedKeySystems.push({ks: supportedKS[ksIdx].ks, configs: [ksConfig]});
+                        requestedKeySystems.push({ks: supportedKS[ksIdx].ks, configs: supportedKS[ksIdx].ks.getKeySystemConfigurations(videoInfo, audioInfo, self.sessionType)});
 
```

Comment:
@bwidtmann Same comments as before...  The special handling is not specific to the key system, it is specific to the EME implementation in a particular browser.  Need to make changes to ProtectionModel.